requirements.txt

```diff
@@ -34,7 +34,6 @@ pymysql-sa==1.0
 pyramid==1.10.4
 python-dateutil==2.8.0
 python-editor==1.0.4
-readline==6.2.4.1
```

Comment:
Not entirely related to this PR, but some self-hosters have reported trouble building `readline` and I don't think it's a strictly necessary dependency.